test/integration/patients/patient/patient-flow.js

```diff
@@ -1323,8 +1327,12 @@ context('patient flow page', function() {
 
     cy
       .routeAction(fx => {
-        fx.data.id = '1';
-        fx.data.relationships.state.data.id = '22222';
+        fx.data = getAction({
```

Comment:
this is a little odd.  seems like it should be `fx.data = testFlowActions[0];` Seems like the relation is already set there.

Reply:
Updated in https://github.com/RoundingWell/care-ops-frontend/pull/1249/commits/dff45dd23f28609ead160543dac53a42dcff8024.